### ISSUE TITLE
Current State of the DMXOPL Project

### ISSUE THREAD

**Issue by sneakernets** [sneakernets/DMXOPL#49]
I realize that this project has been dormant for over a year, mostly due to Real Life ™️ getting in the way. Now that Eternity Engine has ADLMIDI support and life has calmed down considerably, I can begin working on this project again.

A list of things I want to address:

* Credit - DMXOPL currently is a mishmash of contributions. Given the binary nature of the format we use, it's not really easy to track. I tried to keep tabs on this in my commits and release notes, but some uncredited stuff still slipped through. 
* Bloat - There are a lot of files in the repo that are out of date and too large, including MP3 files. I'm not sure if there is a good way to remove these without breaking the repo, but these files will not be updated and may be removed in future.
* DMX support - This was dropped due to constant issues with exporting the WOPL file to the DMX format, which meant that several file formats had to be maintained separately. I want to investigate why these issues cropped up and hopefully fix them.
* Everything in #48 if possible.

**Comment by Wohlstand** [comment]
Oh, maybe you know, on the side of libADLMIDI I have a small news for you that here is a WIP thing which is almost completed: a full rework of embedded banks database which allows storing of multibank and which is better designed which results reduced size of generated store 🦊

**Comment by Wohlstand** [comment]
Oh, maybe you know, on the side of libADLMIDI I have a small news for you that here is a WIP thing which is almost completed: a full rework of embedded banks database which allows storing of multibank and which is better designed which results reduced size of generated store 🦊

**Comment by sneakernets** [comment]
This is great news! thank you!

**Comment by sneakernets** [comment]
Starting this back up, hopefully I can get DMX support back somehow?

**Comment by freq-mod** [comment]
so dead

**Comment by sneakernets** [comment]
> so dead

Nah. I've just been super busy with real life right now. The project isn't going to die any time soon.

**Comment by Wohlstand** [comment]
About the DMX side: as I already explained via Discord, there is a bug at DMX library that incorrectly proceeds AM voices, and I even implemented a simulation of this bug at libADLMIDI and OPL3 Bank Editor sides, when you select the "DMX" volume model. Also, there is the "DMX (Fixed AM)" volume model that runs the DMX volume model without DMX's bug.

**Comment by MoerasGrizzly** [comment]
> About the DMX side: as I already explained via Discord, there is a bug at DMX library that incorrectly proceeds AM voices, and I even implemented a simulation of this bug at libADLMIDI and OPL3 Bank Editor sides, when you select the "DMX" volume model. Also, there is the "DMX (Fixed AM)" volume model that runs the DMX volume model without DMX's bug.

Which volume model would you recommend for DMXOPL playback?
